sensor/FamilyProtectPlugin.js

```diff
@@ -49,6 +51,10 @@ class FamilyProtectPlugin extends Sensor {
             start: this.start,
             stop: this.stop
         });
+        const isPolicyEnabled = await spt.isPolicyEnabled('family');
```

Comment:
this should be one-time only. After this upgrade, if adblock policy:system flag is on and adblock feature flag is off, the feature flag should be kept as off